loqusdbapi/models.py

```diff
@@ -48,7 +48,7 @@ def id_to_str(cls, value):
 
 
 class BaseVariant(BaseModel):
-    id: str = Field(alias="_id")
+    id: Optional[Any] = Field(alias="_id")
```

Comment:
What about having either a string or a Bson object in here?

Reply:
Or actually moving it back to the `Variant` class where it was before the issue. I'll do that!

Reply:
Yes, it makes more sense!